hack/.golint_failures

```diff
@@ -66,6 +66,7 @@ pkg/apis/rbac/v1beta1
 pkg/apis/rbac/validation
 pkg/apis/scheduling
 pkg/apis/scheduling/v1alpha1
```

Comment:
why are you keeping v1alpha1?

Reply:
I think as I understand, it might be needed, if some apis are alpha and some are beta?

Reply:
That is correct. I think we don't have a beta group for scheduling yet. So, we need it say if new api like priorityClass comes up in future.